staff/asier/react-spotify-api-promises/design/scripts/spotify-api-1.0.0.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * Spotify API client.
+ *
+ * @version 1.0.0
+ */
+
+let spotifyApi;
+
+(function () {
+    "use strict";
+    const baseUrl = "https://api.spotify.com/v1/"
+    const token = 'BQCt_qBCV1kEnGnXS9KgyZpo5AJvcwfo3o1gPE6RJGz09CaZSJ9MUMd1pAjNnV8JxGH2kPMhZqQaNLdWGyGSYAyTgsQLBkBQjhtGe2kFph1PF_Rai6cPo-4k5nRnT4pbj-OJVAw'

```

Comment:
WARN token is a settable part of spotifyApi, and should be accessible from it to update it:

spotifyApi.token = ...

otherwise, if defining it as a scoped reference, it forces the user of spotifyApi to modify the spotifyApi.js file, something not desirable, neither convenient.

so token should be a member property of spotifyApi, but not a private scoped ref.